src/Impl/GuzzleHttpRequester.php

```diff
@@ -85,7 +88,8 @@ private function parseMessageStatus(ResponseInterface $response): MessageStatus
     private function parseFailureResponse(ResponseInterface $response): FailureResponse
     {
         try {
-            $jsonObject = $this->parseResponseToJson($response);
+            $jsonString = (string)$response->getBody();
+            $jsonObject = $this->parseToJsonObject($jsonString, $response);
             return FailureResponse::fromJson($jsonObject);
```

Comment:
No change in FailureResponse in the ingestion service - it is still being returned on _severe_ error - along with specific error status:
Unchanged code:
```php
        if (in_array($statusCode, [400, 403, 500])) {
            $failureResponse = $this->parseFailureResponse($response);
            throw $this->streamxClientExceptionFrom($failureResponse);
        }
```

Otherwise, the service returns list of SuccessResult or FailureResponse